src/main/java/org/spdx/tools/Verify.java

```diff
@@ -95,9 +94,9 @@ public static void main(String[] args) {
 			System.exit(ERROR_STATUS);
 		}
 		// separate out the warning from errors
-		List<String> warnings = new ArrayList<>();
-		List<String> errors = new ArrayList<>();
-		for (String verifyMsg:verify) {
+		List<String> warnings = new ArrayList<String>();
```

Comment:
```suggestion
		List<String> warnings = new ArrayList<>();
```

I believe it is a preferred coding style to not specify the type for a generic in the constructor

Reply:
Thank you. I revert the changes back.